README.md

```diff
@@ -1,69 +1,32 @@
 ## Configure Development Environment
 
 ### Install dev/build tools
-- Node.js 16.x or higher
 - [Yarn](https://yarnpkg.com)
+- Node.js 16.x or higher (installed with yarn)
 
 ### Create dev database
-Development is supported on either a local MongoDB server or an account on [Mongo Atlas](https://www.mongodb.com/atlas/database).
-
-#### Local MongoDB Server
-For a local installation, it's recommended to install the [Community Server](https://www.mongodb.com/try/download/community) and [Compass UI](https://www.mongodb.com/try/download/compass).
-
-Add the connection string for your server and database to `.env.local`:
-```
-MONGODB_URI="mongodb://localhost:27017/respond-dev?retryWrites=true&w=majority"
-```
-
-#### MongoDB Atlas service:
-Add the connection string for your database to `.env.local`:
-```
-MONGODB_URI="mongodb+srv://database:<your-connection-string>.mongodb.net/respond-dev?retryWrites=true&w=majority"
-```
+Development is supported on either a local or remote MongoDB server.
+- For a local installation, it's recommended to install the [Community Server](https://www.mongodb.com/try/download/community) and [Compass UI](https://www.mongodb.com/try/download/compass).
+- For a remote DB, use [Mongo Atlas](https://www.mongodb.com/atlas/database). You can also use [Compass UI](https://www.mongodb.com/try/download/compass), but isn't strictly necessary. 
 
+In your MongoDB, add a database named `respond-dev`. Then also add a user for that database. In Mongo Atlas this is done by clicking on "Database Access" from the left side bar navigation. Then click the "Add new database user" and give them read/write permission. This username/password is what you'll use in your connection string below. 
```

Comment:
nit: Adding a user is only necessary for a remote database.

Reply:
Updated.